tests/data/config/non_valid_toml_boolopt/pyproject.toml

```diff
@@ -3,4 +3,5 @@ write = false
 style = ["numpydoc"]
 strip-whitespaces = true
 split-summary-body = false
+closing-quotes = false
```

Comment:
I'd prefer updating the example code.

Reply:
As it is right now it is not possible, those two are not compatible. (numpydoc and closing quotes, because numpydoc enables NumpydocSectionSpacingFormatter)

I think out loud about it here:
https://github.com/DanielNoord/pydocstringformatter/pull/173#issuecomment-1249930115

Reply:
We probably need to fix #177 before we can merge this PR.

Reply:
i think this is mergeable and we can have 177 be part of the milestones for the next release

Reply:
I'm removing these settings as I have merged #181 and they are now unnecessary 🎉